content/en/docs/tasks/inject-data-application/define-environment-variable-container.md

```diff
@@ -84,6 +84,11 @@ The environment variables set using the `env` or `envFrom` field
 override any environment variables specified in the container image.
 {{< /note >}}
 
+{{< note >}}
+The environment variables can reference each other, and cycles are possible,
+pay attention to the order before using
```

Comment:
```suggestion
The environment variables can reference each other, and cycles are possible,
pay attention to the order before using.
```

If I create a cycle, what happens? Does the Pod launch OK? Does it _make demons fly out of my nose_? I think the reader would want the behavior defined (possibly on another page, but defined somewhere).

I also think it's OK to merge this now and log an issue to track the work to define that behavior.

Reply:
Pods can be started. If a circular dependency occurs, the final value of the environment variable may not be what we expected. I suggest that you can open a new page to explain this, or give some common examples. This pr can be merged first